values/prometheus-operator/prometheus-operator.gotmpl

```diff
@@ -156,6 +156,18 @@ prometheus:
 additionalPrometheusRules:
   - name: blackbox
     {{- readFile "rules/blackbox.yaml" | nindent 4 }}
+  {{ if ($v.platformBackups.database.gitea | get "enabled" false) }}
```

Comment:
From the defaults.yaml
```
        platformBackups:
          database:
            harbor:
              enabled: false
            gitea:
              enabled: false
            keycloak:
              enabled: false
```

you can safely refer to  `$v.platformBackups.database.gitea.enabled`

Reply:
Habit. Adjusted.

Reply:
thanks